Crash report 2.9.1: NPE in DisplayUtils.getDisplayPixelSize

```
java.lang.RuntimeException: An error occured while executing doInBackground()
       at android.os.AsyncTask$3.done(AsyncTask.java:278)
       at java.util.concurrent.FutureTask$Sync.innerSetException(FutureTask.java:273)
       at java.util.concurrent.FutureTask.setException(FutureTask.java:124)
       at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:307)
       at java.util.concurrent.FutureTask.run(FutureTask.java:137)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1076)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:569)
       at java.lang.Thread.run(Thread.java:856)
Caused by: java.lang.NullPointerException
       at org.wordpress.android.util.DisplayUtils.getDisplayPixelSize(DisplayUtils.java:28)
       at org.wordpress.android.util.DisplayUtils.getDisplayPixelWidth(DisplayUtils.java:36)
       at org.wordpress.android.ui.reader.ReaderPostDetailFragment.getFullSizeImageWidth(ReaderPostDetailFragment.java:1035)
       at org.wordpress.android.ui.reader.ReaderPostDetailFragment.getPostHtml(ReaderPostDetailFragment.java:1071)
       at org.wordpress.android.ui.reader.ReaderPostDetailFragment.access$3000(ReaderPostDetailFragment.java:68)
       at org.wordpress.android.ui.reader.ReaderPostDetailFragment$ShowPostTask.doInBackground(ReaderPostDetailFragment.java:1265)
       at org.wordpress.android.ui.reader.ReaderPostDetailFragment$ShowPostTask.doInBackground(ReaderPostDetailFragment.java:1217)
       at android.os.AsyncTask$2.call(AsyncTask.java:264)
       at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:305)
       at java.util.concurrent.FutureTask.run(FutureTask.java:137)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1076)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:569)
       at java.lang.Thread.run(Thread.java:856)
```

--- a/src/org/wordpress/android/ui/reader/ReaderPostDetailFragment.java
+++ b/src/org/wordpress/android/ui/reader/ReaderPostDetailFragment.java
@@ -973,8 +973,8 @@ public class ReaderPostDetailFragment extends Fragment
     /*
      * size to use for images that fit the full width of the listView item
      */
-    private int getFullSizeImageWidth() {
-        int displayWidth = DisplayUtils.getDisplayPixelWidth(getActivity());
+    private int getFullSizeImageWidth(Context context) {
+        int displayWidth = DisplayUtils.getDisplayPixelWidth(context);
         int marginWidth = getResources().getDimensionPixelOffset(R.dimen.reader_list_margin);
         int imageWidth = displayWidth - (marginWidth * 2);
         if (hasStaticMenuDrawer()) {
@@ -1011,7 +1011,7 @@ public class ReaderPostDetailFragment extends Fragment
         int marginLarge = context.getResources().getDimensionPixelSize(R.dimen.margin_large);
         int marginSmall = context.getResources().getDimensionPixelSize(R.dimen.margin_small);
         int marginExtraSmall = context.getResources().getDimensionPixelSize(R.dimen.margin_extra_small);
-        int fullSizeImageWidth = getFullSizeImageWidth();
+        int fullSizeImageWidth = getFullSizeImageWidth(context);
 
         final String linkColor = HtmlUtils.colorResToHtmlColor(context, R.color.reader_hyperlink);
         final String greyLight = HtmlUtils.colorResToHtmlColor(context, R.color.grey_light);
@@ -1217,7 +1217,7 @@ public class ReaderPostDetailFragment extends Fragment
                     showFeaturedImage = true;
                     // note that only the width is used here - the imageView will adjust
                     // the height to match that of the image once loaded
-                    featuredImageUrl = mPost.getFeaturedImageForDisplay(getFullSizeImageWidth(), 0);
+                    featuredImageUrl = mPost.getFeaturedImageForDisplay(getFullSizeImageWidth(container.getContext()), 0);
                 }
             }
 

